### PR TITLE
5512/pr tests no userzone

### PR DESCRIPTION
**Pull request by sblack-usu**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

File: .github/workflows/hydroshare-pull-requests.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - uses: actions/checkout@v2
+
+    - name: Install Docker Compose
+      run: |
+        sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+        sudo chmod +x /usr/local/bin/docker-compose
+
+    - name: Setup HydroShare
+      run: >
+        printf "%s\n" c | ./local-dev-first-start-only.sh
```
Comment (devincowan) [inline .github/workflows/hydroshare-pull-requests.yml:28]: this is awesome! So simple, I love it!

**Review by devincowan — APPROVED** [review]

File: .github/workflows/hydroshare-pull-requests.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      run: >
+        docker-compose -f local-dev.yml up -d
+
+    - name: Wait for HydroShare to come up
+      run: >
+        sleep 60
+
+    - name: Run HydroShare Tests
+      run: >
+        docker exec hydroshare ./run-tests-jenkins
```
Comment (devincowan) [inline .github/workflows/hydroshare-pull-requests.yml:40]: need to add failure detection I guess

**Comment by devincowan** [comment]
I see you've used https://github.com/coroo/pytest-coverage-commentator before
I wonder if https://github.com/EnricoMi/publish-unit-test-result-action could do what we want

**Comment by devincowan** [comment]
that `publish-unit-test-result-action` produces reports like this:
https://github.com/hydroshare/hydroshare/runs/27018450778

**Comment by sblack-usu** [comment]
Closing, this has been merged into [manual-irods-userzone-and-rm-federation](https://github.com/hydroshare/hydroshare/tree/manual-irods-userzone-and-rm-federation)

**Pull request closed without merging**